### DISCUSSION TITLE
Hide or Password Protect words and sections in documents

### DISCUSSION THREAD

**Discussion by julianbrennum** [outline/outline#6160] [Ideas]
I think it would be nice if we could hide or password protect some parts of a document. 

When writing documentation there could be some sensitive information that should not be visible without clicking on the "eye" icon like in a password manager or hover over the text. I don't know if there is any need for it to be password protected, but at least have it to be hidden, so when you open some guides or something you don't expose the "sensitive" fields right away.

This could be useful when browsing a document in public or sharing it via screenshare in a meeting etc. Then some fields should stay hidden unless you hover/click on it to reveal it.

**Comment by tommoor** [comment]
I would be wary of adding encryption or password protection here, I think it would encourage people to store passwords which isn't wise. However perhaps some sort of redacted mark that only shows the content when you focus could be quite interesting

**Comment by kvalter** [comment]
Yes, still needed. Please add this feature

**Comment by wangweiningse** [comment]
Yes, still needed.
